cmd/cronosd/cmd/root.go

```diff
@@ -57,7 +57,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 
 	encodingConfig := app.MakeEncodingConfig()
 	initClientCtx := client.Context{}.
-		WithJSONCodec(encodingConfig.Marshaler).
+		WithCodec(encodingConfig.Marshaler).
```

Comment:
are those the same?

Reply:
`WithJSONCodec` has a deprecated warning.